### PR TITLE
fix: ensure calico cni dir

### PR DESCRIPTION
**Pull request by angao**
**What type of PR is this?**
/kind bug
/area network

**What this PR does / why we need it**:
- 确保 calico cni 目录存在

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hanxueluo @WangMinjie114 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: roles/network_plugin/bd-calico/tasks/pre.yml [hunk 1]
```diff
@@ -9,3 +9,9 @@
   delegate_to: "{{ groups['kube-master'][0] }}"
   when:
     - "cloud_provider is defined"
+
+- name: Calico | Ensure calico cni dir
```
Comment (hanxueluo) [inline roles/network_plugin/bd-calico/tasks/pre.yml]: 这应该写在 multus 那边吧

Comment (angao) [reply roles/network_plugin/bd-calico/tasks/pre.yml]: 先执行的 calico task

Comment (hanxueluo) [reply roles/network_plugin/bd-calico/tasks/pre.yml]: ？这么奇怪？那当前顺序有问题啊。

Comment (angao) [reply roles/network_plugin/bd-calico/tasks/pre.yml]: kubespray 里默认的顺序

Comment (hanxueluo) [reply roles/network_plugin/bd-calico/tasks/pre.yml]: 应该改这个文件:
roles/kubernetes/preinstall/tasks/0050-create_directories.yml

**Review by bbbmj — COMMENTED** [review]
/lgtm

**Pull request merged**